Userland/Libraries/LibAudio/ClientConnection.cpp

```diff
@@ -20,7 +20,7 @@ void ClientConnection::enqueue(const Buffer& buffer)
         auto success = enqueue_buffer(buffer.anonymous_buffer(), buffer.id(), buffer.sample_count());
         if (success)
             break;
-        sleep(1);
+        sleep(.1);
```

Comment:
This is just `sleep(0)`:
```c++
unsigned int sleep(unsigned int seconds);
```
You want to use `usleep(useconds_t)` (i.e. `usleep(100000);`)

Reply:
Ah thanks, I missed that.